tests/conftest.py

```
@@ -116,6 +117,194 @@ class MockRequestsExceptions:
+            result_fn()
+    except (
+        TimeoutError,
+        asyncio.TimeoutError,
+        asyncio.CancelledError,
+        asyncio.InvalidStateError,
+        concurrent.futures.TimeoutError,
+        concurrent.futures.CancelledError,
+        concurrent.futures.InvalidStateError,
+    ):
+        return
+    except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)